java/org/apache/jasper/EmbeddedServletOptions.java

```diff
@@ -84,6 +84,11 @@
      */
     private boolean classDebugInfo = true;
 
+    /**
+     * Do we want to Trim White Space in the in JSP Files?
```

Comment:
why `Trim White Space` is capitalized here ?

Reply:
Resolved with latest push